lib/modules/stylesheet.js

```diff
@@ -222,46 +223,49 @@ addModule('stylesheet', function(module, moduleID) {
 		RESUtils.bodyClasses.remove.apply(RESUtils, removeClasses);
 	}
 
-
-	function sanitizeSubredditList(uncleanRows) {
-		var uncleanValues = uncleanRows.map(function(row) {
-				return row[0];
-			});
-
-		var subreddits = RESUtils.options.listTypes['subreddits'].sanitizeValues(uncleanValues);
-		return subreddits;
+	var subredditNameRegexp = /^(?:\/?r\/)?([\w_]+)\/?$/;
+	var urlRegexp = /^(?:https?:\/\/[\w\.]+)?\/\w+/;
+	function sanitizeStylesheetUrls(rows) {
+		return rows.map(function(row) {
+			var url = row[0];
+			if (subredditNameRegexp.test(url)) {
+				return document.location.origin + '/r/' + subredditNameRegexp.exec(url)[1] + '/stylesheet.css';
```

Comment:
Oh, yeah, I incorrectly thought about referencing assets inside the extension -- but it's nbd because injected link